packages/core/src/CSL/cslToCore/cslToCore.ts

```diff
@@ -1,6 +1,5 @@
 import { Asset, CSL, Cardano, SerializationFailure, util } from '../..';
-import { BootstrapWitness } from '@cardano-ogmios/schema';
-import { Ed25519KeyHash, NativeScriptType } from '../../Cardano';
+import { Ed25519KeyHash, NativeScriptKind, ScriptType, typedHex } from '../../Cardano';
```

Comment:
Not an issue with this PR as it was already there, but we already have an import of the entire Cardano namespace
```suggestion
```

Reply:
Done in 5f8f046abf806f189c96be63ca12934907da27bc